cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -43,6 +44,43 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     cy.login();
   });
 
+  // testing https://github.com/rancher/dashboard/issues/9823
+  it('Toggling the user preference "rke1-ui" to false should not display RKE toggle on cluster creation screen and hide RKE1 resources from nav', () => {
+    cy.intercept('GET', 'v1/management.cattle.io.features?*', {
```

Comment:
since we're testing the functionality of the feature flag, can we toggle the feature flag in the UI instead of using intercept here?

Reply:
That is not possible since the feature flag doesn't exist on the backend yet as per https://github.com/rancher/rancher/issues/43131. Me and @nwmac decided to move this forward as it was low-risk (behind a flag) work that was still hanging since our colleague left.

Reply:
I'm referring to the front end. We can't toggle it from the front end? maybe I'm missing something

Reply:
No, we can't toggle it from the frontend. Since the flag doesn't exist on the backend, we don't have it available on the interface to toggle it. Feature flags can exist (if I am not mistaken) without even being exposed to the frontend. 

The ones that are exposed can be found under -> `Global Settings` -> `Feature Flags`.

Reply:
Ah I see thank you for clarifying that